scripts/openshiftci-presubmit-unittests.sh

```diff
@@ -11,7 +11,7 @@ export PATH=$PATH:$GOPATH/bin
 # otherwise /.cache is used, and it fails on permission denied
 export GOLANGCI_LINT_CACHE="/tmp/.cache"
 
-make goget-tools
+# make goget-tools
```

Comment:
Is this temporary?

Reply:
All these changes are just to trial and error to find out why it's failing on CI and working fine on local. And also we will try to use a specific version to avoid any surprises.